Cargo.toml

```diff
@@ -61,6 +61,7 @@ version = "0.3.0"
 
 [build-dependencies]
 lazy_static = "1.1.0"
+embed-resource = "1.6"
```

Comment:
I don't think this is needed for non-Windows builds right? Could you please make this a target specific dependency for windows only? Thanks!